tsdb/engine/tsm1/cache.go

```diff
@@ -101,22 +88,11 @@ func (e *entry) add(values []Value) error {
 		if e.vtype != valueType(v) {
 			return tsdb.ErrFieldTypeConflict
 		}
-
```

Comment:
Since we're no longer checking if the values are sorted we can alter the comment about them being "out of order" on line 86.